app/gui/src/dashboard/components/Spinner.tsx

```diff
@@ -63,7 +63,7 @@ export const Spinner = React.memo(function Spinner(props: SpinnerProps) {
         strokeLinecap="round"
         strokeWidth={thickness}
         className={twJoin(
-          'pointer-events-none origin-center !animate-spin-ease transition-stroke-dasharray [transition-duration:var(--spinner-slow-transition-duration)]',
```

Comment:
@MrFlashAccount @somebody1234 I removed it, because it effectively overrode the information from state. If it was a conscious decision, let me know and I restore it.

Reply:
If the animation remains the same then I'm fine with the solution

Reply:
Well, it's not the same: before, the animation was always slow, regardless if the passed state was `loading-fast`, `loading-medium` or `loading-slow`. I assumed this was an overlooked bug.